bundles/ch.itmed.fop.printing.ui/src/ch/itmed/fop/printing/handlers/ArticleMedicationLabelsHandler.java

```diff
@@ -99,4 +109,12 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 		}
 		return null;
 	}
+    private static String getDosageInstructions(IArticle article) {
```

Comment:
Hier wäre es gut Optional zu verwenden. `Optional<String> getDosageInstruction(IArticle article)`

dann `return Optional.of(signature.getComment())` bzw. `return Optional.empty()`